python/arm.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+"""Lifted ARM instruction"""
+
+from adt import *
+from asm import *
+from bil import *
+
+class Reg(ADT) : pass
+class Nil(Reg) : pass
+class GPR(Reg) : pass
+class CCR(Reg) : pass
+
+class R0(GPR) : pass
+class R1(GPR) : pass
+class R2(GPR) : pass
+class R3(GPR) : pass
+class R4(GPR) : pass
+class R5(GPR) : pass
+class R6(GPR) : pass
+class R7(GPR) : pass
+class R8(GPR) : pass
+class R9(GPR) : pass
+class R10(GPR) : pass
+class R11(GPR) : pass
+class R12(GPR) : pass
+class LR(GPR) : pass
+class PC(GPR) : pass
+class SP(GPR) : pass
+
+class CPSR(CCR) : pass
+class SPSR(CCR) : pass
+class ITSTATE(CCR) : pass
+
+class Insn(ADT)     : pass
+class Move(Insn)    : pass
+class Bits(Insn)    : pass
+class Mult(Insn)    : pass
+class Mem(Insn)     : pass
+class Branch(Insn)  : pass
+class Special(Insn) : pass
+
+class ADCri(Move) : pass
```

Comment:
Where's the script to autogenerate this? If there isn't one, there should be. If there is one, it should be indicated in a comment int his file so people don't edit it.